### ISSUE TITLE
Black Explorer navigation pane in latest actions build with themes without Mica

### ISSUE THREAD

**Issue by Excremation** [Rectify11/Installer#519]
### Description

In the latest actions build, the left hand navigation pane of Explorer has a solid black background when using any Rectify11 dark theme other than "Dark theme with Mica".

I have downgraded back to the official 3.1 release, and it displays correctly (see screenshot).
Apologies I can't attach a screenshot of it on the newer build without updating to it all over again, but I have highlighted the referenced area in the screenshot.

I'm using Rectify11 without MicaForEveryone, paired with DWMBlurGlass v2.1.1 and ExplorerBlurMica v2.0.1.

### Steps To Reproduce

Install the latest actions build of R11 with DWMBlurGlass v2.1.1 and ExplorerBlurMica v2.0.1, enable any of the dark themes without Mica.

### Rectify11 Version

Latest actions build, #794

### Windows Version

Windows 11 23H2, build 22631.3447

### Logs

![Explorer R11 v3 1](https://github.com/Rectify11/Installer/assets/136895206/6c5bbeff-960f-49df-85f0-7fd007b4715d)
![Explorer R11 Highlighted Area](https://github.com/Rectify11/Installer/assets/136895206/8fc1126a-5594-4870-8e13-25edba96087b)

**Comment by CinceTheDay** [comment]
I can confirm this
![k6lEjSwwgm](https://github.com/Rectify11/Installer/assets/162928034/b5995869-cdbc-4f4d-80ac-c85b39e9477c)

**Comment by Excremation** [comment]
> I can confirm this
> ![k6lEjSwwgm](https://private-user-images.githubusercontent.com/162928034/335201247-b5995869-cdbc-4f4d-80ac-c85b39e9477c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcwNzY3ODgsIm5iZiI6MTcxNzA3NjQ4OCwicGF0aCI6Ii8xNjI5MjgwMzQvMzM1MjAxMjQ3LWI1OTk1ODY5LWNkYmMtNGY0ZC04MGFjLWM4NWIzOWU5NDc3Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUzMFQxMzQxMjhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04NTJlYWIzNjUxODgzMTAyMjE2MTFhZDU0OGVlYTZmYzIwNWFjOThhOWEzNzM4MDU2N2EyYjU5ODJhN2E0OTA1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.gD7oRA8E5ttcHo_5kicqtVPI0CMqctpct14aO33S3G8)

Yep this is exactly it, thanks for providing your screenshot where I couldn’t.
